tests/conftest.py

```diff
@@ -133,6 +133,31 @@ def clean(dsw_sdk):
     _clean(dsw_sdk)
 
 
+# ----------------------------- Questionnaires -------------------------------
+
+
+@pytest.fixture
+def qtn_data(request):
+    data = {
+        'name': 'My questionnaire',
+        'packageId': 'dsw:root:2.3.0',
```

Comment:
There is initially no package in the DSW. So we should first create the package with `package` fixture and use its ID here.

Be aware though, that there is an automatic cleanup (the `clean` fixture) deleting entities from fixtures in particular order. But the questionnaire has to contain `test` keyword in its name in order to get deleted.